src/data/finset/basic.lean

```diff
@@ -2864,9 +2861,6 @@ disjoint_of_subset_right (inter_subset_right _ _) sdiff_disjoint
 lemma sdiff_eq_self_iff_disjoint {s t : finset α} : s \ t = s ↔ disjoint s t :=
 by rw [sdiff_eq_self, subset_empty, disjoint_iff_inter_eq_empty]
 
-lemma sdiff_eq_self_of_disjoint {s t : finset α} (h : disjoint s t) : s \ t = s :=
```

Comment:
What's the purpose of removing this lemma? It's nice to be able to rewrite `s \ t` into `s` given a `disjoint s t` rather than needing to use `sdiff_eq_self_iff_disjoint.2` which is less readable. We also have the mathlib pattern more generally of giving `of` versions of `iff` lemmas, so this isn't unique to this file.

Reply:
This lemma (and the above and even more) is a `finset` duplicate of [the generalized boolean  algebra one](https://leanprover-community.github.io/mathlib_docs/order/boolean_algebra.html#disjoint.sdiff_eq_left). I got confused when writing this PR, so I just straight up deleted it, but I can of course do that (and the other ones) in a separate PR.

Reply:
I'll delete them in another PR.